Chatto/Source/ChatController/BaseChatViewController+Scrolling.swift

```diff
@@ -36,79 +36,97 @@ extension CGFloat {
 extension BaseChatViewController {
 
     public func isScrolledAtBottom() -> Bool {
-        guard self.collectionView.numberOfSections > 0 && self.collectionView.numberOfItems(inSection: 0) > 0 else { return true }
-        let sectionIndex = self.collectionView.numberOfSections - 1
-        let itemIndex = self.collectionView.numberOfItems(inSection: sectionIndex) - 1
+        guard
+            let collectionView = self.collectionView,
+            collectionView.numberOfSections > 0
+                && collectionView.numberOfItems(inSection: 0) > 0
+        else { return true }
+        let sectionIndex = collectionView.numberOfSections - 1
+        let itemIndex = collectionView.numberOfItems(inSection: sectionIndex) - 1
         let lastIndexPath = IndexPath(item: itemIndex, section: sectionIndex)
         return self.isIndexPathVisible(lastIndexPath, atEdge: .bottom)
     }
 
     public func isScrolledAtTop() -> Bool {
-        guard self.collectionView.numberOfSections > 0 && self.collectionView.numberOfItems(inSection: 0) > 0 else { return true }
+        guard
+            let collectionView = self.collectionView,
+            collectionView.numberOfSections > 0
+                && collectionView.numberOfItems(inSection: 0) > 0
+        else { return true }
         let firstIndexPath = IndexPath(item: 0, section: 0)
         return self.isIndexPathVisible(firstIndexPath, atEdge: .top)
     }
 
     public func isCloseToBottom() -> Bool {
-        guard self.collectionView.contentSize.height > 0 else { return true }
-        return (self.visibleRect().maxY / self.collectionView.contentSize.height) > (1 - self.constants.autoloadingFractionalThreshold)
+        guard
+            let collectionView = self.collectionView,
+            collectionView.contentSize.height > 0
+        else { return true }
+        return (self.visibleRect().maxY / collectionView.contentSize.height) > (1 - self.constants.autoloadingFractionalThreshold)
     }
 
     public func isCloseToTop() -> Bool {
-        guard self.collectionView.contentSize.height > 0 else { return true }
-        return (self.visibleRect().minY / self.collectionView.contentSize.height) < self.constants.autoloadingFractionalThreshold
+        guard
+            let collectionView = self.collectionView,
+            collectionView.contentSize.height > 0
+            else { return true }
+        return (self.visibleRect().minY / collectionView.contentSize.height) < self.constants.autoloadingFractionalThreshold
     }
 
     public func isIndexPathVisible(_ indexPath: IndexPath, atEdge edge: CellVerticalEdge) -> Bool {
-        if let attributes = self.collectionView.collectionViewLayout.layoutAttributesForItem(at: indexPath) {
+        guard
+            let collectionView = self.collectionView,
+            let attributes = collectionView.collectionViewLayout.layoutAttributesForItem(at: indexPath)
+        else { return false }
             let visibleRect = self.visibleRect()
```

Comment:
Those lines have wrong indentation after `if` statement was removed.

Reply:
Sorry, It happens when swiftformat fixes it for you on the main project.